docs/docs/dev-server/writing-plugins/hooks.md

```diff
@@ -332,6 +332,10 @@ The `serverStart` hook is called when the server starts. It is the ideal locatio
 Accessing the serverStart parameters:
 
 ```js
+import _glob from 'glob';
```

Comment:
FYI: https://www.npmjs.com/package/fast-glob does this without the promise wrapping, not sure if it much the same otherwise or not.

LGTM, otherwise.

Reply:
yeah, `globby` as well

I figured `glob` is a 'lowest-common-denominator' kind of package

Reply:
If wrapping is fine, seems fine. I’ve never wrapped like that to say if it feels “normal” or not.